### PR TITLE
Static Pages

### PR DESCRIPTION
**Pull request by brokenheart1712**

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <ul>
+      <li>
+        <%= link_to "About", root_path %>
+      </li>
+      <li>
+        <%= link_to "Contact", root_path %>
+      </li>
+    </ul>
+  </nav>
+</footer>
```
Comment (NguyenTanDuc) [inline app/views/layouts/_footer.html.erb]: Add new line in end of file

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <li>
+          <%= link_to "Result", root_path %>
+        </li>
+        <li>
+          <%= link_to "User Show", root_path %>
+        </li>        
+      </ul>
+    </nav>
+  </div>
+</header>
```
Comment (NguyenTanDuc) [inline app/views/layouts/_header.html.erb]: Add new line in end of file

File: app/views/layouts/_shim.html.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+  </script>
+<![endif]-->
```
Comment (NguyenTanDuc) [inline app/views/layouts/_shim.html.erb]: Add new line in end of file

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,14 +1,23 @@
...
+			<% flash.each do |message_type, message| %>
+			<div class="alert alert-<%= message_type %>">
+				<%= message %>
+			</div>
+			<% end %>
+			<%= yield %>
+			<%= render 'layouts/footer' %>
+		</div>
+	</body>
+</html>
```
Comment (NguyenTanDuc) [inline app/views/layouts/application.html.erb]: Add new line in end of file

File: app/views/static_pages/home.html.erb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<div class="center jumbotron">
+  <h1>Welcome to the Framgia E-Learning System!</h1>
+  <h2> This is the home page for the Framgia E-Learning System. </h2>
+  <%= link_to "Sign Up Now!", root_path, class: "btn btn-lg btn-primary" %>
+</div>
```
Comment (NguyenTanDuc) [inline app/views/static_pages/home.html.erb]: Add new line in end of file

File: app/assets/stylesheets/application.css.scss [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+    margin-bottom: 5px;
+  }
+}
+
+span.picture {
+  margin-top: 10px;
+  input {
+    border: 0;
+  }
+}
```
Comment (NguyenTanDuc) [inline app/assets/stylesheets/application.css.scss]: Add new line in end of file

**Comment by NguyenTanDuc** [comment]
11 file changed

**Comment by NguyenTanDuc** [comment]
It's OK.

**Comment by NguyenTanDuc** [comment]
No problem now.

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -1,14 +1,22 @@
...
+    <%= stylesheet_link_tag "application", media: "all",
+    "data-turbolinks-track" => true %>
+    <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+    <%= csrf_meta_tags %>
+  </head>
+  <body>
+    <%= render 'layouts/header' %>
+    <div class="container">
+      <% flash.each do |message_type, message| %>
+      <div class="alert alert-<%= message_type %>">
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: indent 
```
+      <div class="alert alert-<%= message_type %>">
+        <%= message %>
+      </div>
```

**Comment by NguyenTanDuc** [comment]
OK.

**Pull request merged**